### PR TITLE
Generate tasks.json taxonomy from `huggingface_hub`

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

File: src/datasets/utils/resources/tasks.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
         "subtasks": [
             "keyword-spotting",
             "speaker-identification",
-            "audio-intent-classification",
-            "audio-emotion-recognition",
-            "audio-language-identification"
+            "speaker-intent-classification",
+            "emotion-recognition",
+            "speaker-language-identification"
```
Comment (julien-c) [inline src/datasets/utils/resources/tasks.json:9]: i think those should prob be updated in `huggingface_hub` @osanseviero

Comment (osanseviero) [reply src/datasets/utils/resources/tasks.json:9]: Done in https://github.com/huggingface/hub-docs/pull/95/commits/18c70dac6bb83cfa56d5e7594f5ca45cefbe7049

**Comment by julien-c** [comment]
Ok recomputed the json file, this should be ready to review now! @lhoestq

**Comment by julien-c** [comment]
Note: the generated JSON from `hf/hub-docs` can be found in the output of a GitHub Action run on that repo, for instance in https://github.com/huggingface/hub-docs/runs/6006686983?check_suite_focus=true

(click on "Run export-tasks script")

**Review by lhoestq — APPROVED** [review]
Thanks !

**Comment by osanseviero** [comment]
Should we not add the tasks with hideInDatasets?

**Review by osanseviero — APPROVED** [review]
LGTM! 

(nit, I see `unconditional-image-generation` is here. Are you ok with that @lhoestq? If no, we should add `hideInDatasets` in https://github.com/huggingface/hub-docs/blob/main/js/src/lib/interfaces/Types.ts

**Comment by lhoestq** [comment]
Yes that's good :) feel free to merge

**Comment by julien-c** [comment]
thanks to the both of you!

**Pull request merged**